src/components/icons/ResetFiltersIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const ResetFiltersIcon = createIcon({
```

Comment:
It does not necessarily have to do with filters. Could be used in other places. Let's call it `ResetIcon`. Or align with Figma and call it `RotateIcon`